testing/file_test/autoupdate.cpp

```diff
@@ -278,53 +292,52 @@ auto AutoupdateFileTest(
       // STDOUT check lines are placed after the line they refer to, or at the
       // end of the file if none of them refers to a line.
       if (reached_autoupdate && any_attached_stdout_lines) {
-        add_check_lines(stdout_check_file, stdout_check_line,
+        add_check_lines(stdout_check_lines, stdout_check_line,
                         non_check_line.line_number(), non_check_line.indent());
       }
     }
 
     // This should always be true after the first file is processed.
     CARBON_CHECK(reached_autoupdate);
 
-    // Print remaining check lines which -- for whatever reason -- come after
-    // all original lines.
-    if (stderr_check_line != stderr_check_file.end() ||
-        stdout_check_line != stdout_check_file.end()) {
+    // At the end of the last file, print remaining check lines which -- for
+    // whatever reason -- come after all original lines.
+    if (file_number == filenames.size() - 1 &&
```

Comment:
Just to note, with the shift in structure, I'm starting to think some edits could shift this and the RemapLineNumbers out of the loop.

Reply:
Sure, that's fine, didn't really think it made sense here since as you note the file mappings would be needed.